### ISSUE TITLE
rancher2 tf provider does not recognize the latest 1.15.x k8s version for Rancher 2.3.6

### ISSUE THREAD

**Issue by rosskirkpat** [rancher/terraform-provider-rancher2#299]
When trying to provision a new Rancher 2.3.6 cluster using the TF provider, the latest k8s version available for 1.15.x is `1.15.11-rancher1-2` but it is not a recognized version. Error included below

```
Error:

Error: expected rke_config.0.kubernetes_version to be one of [v1.17.4-rancher1-3 v1.17.4-rancher1-1 v1.17.3-rancher1-1 v1.17.2-rancher1-2 v1.17.2-rancher1-1 v1.17.0-rancher1-2 v1.17.0-rancher1-1 v1.16.8-rancher1-3 v1.16.8-rancher1-1 v1.16.7-rancher1-1 v1.16.6-rancher1-2 v1.16.6-rancher1-1 v1.16.4-rancher1-1 v1.16.3-rancher1-1 v1.16.2-rancher2-1 v1.16.2-rancher1-1 v1.16.1-rancher1-1 v1.15.11-rancher1-3 v1.15.11-rancher1-1 v1.15.11-rancher1-0 v1.15.10-rancher1-1 v1.15.9-rancher1-1 v1.15.7-rancher1-1 v1.15.6-rancher1-2 v1.15.5-rancher2-2 v1.15.5-rancher1-2 v1.15.5-rancher1-1 v1.15.4-rancher1-2 v1.15.4-rancher1-1 v1.15.3-rancher1-1 v1.15.2-rancher1-1 v1.15.0-rancher1-1 v1.14.10-rancher1-1 v1.14.9-rancher1-2 v1.14.9-rancher1-1 v1.14.8-rancher2-1 v1.14.8-rancher1-1 v1.14.7-rancher1-1 v1.14.6-rancher1-1 v1.14.5-rancher1-1 v1.14.3-rancher1-1 v1.14.1-rancher1-2 v1.14.1-rancher1-1 v1.13.12-rancher2-1 v1.13.12-rancher1-1 v1.13.11-rancher1-1 v1.13.10-rancher1-2 v1.13.10-rancher1-1 v1.13.9-rancher1-2 v1.13.9-rancher1-1 v1.13.7-rancher1-1 v1.13.5-rancher1-3 v1.13.5-rancher1-2 v1.13.5-rancher1-1 v1.13.4-rancher1-2 v1.13.4-rancher1-1 v1.13.1-rancher1-2 v1.13.1-rancher1-1 v1.12.10-rancher1-2 v1.12.10-rancher1-1 v1.12.9-rancher1-1 v1.12.7-rancher1-4 v1.12.7-rancher1-3 v1.12.7-rancher1-2 v1.12.7-rancher1-1 v1.12.6-rancher1-2 v1.12.6-rancher1-1 v1.12.5-rancher1-2 v1.12.5-rancher1-1 v1.12.4-rancher1-1 v1.12.3-rancher1-1 v1.12.1-rancher1-1 v1.12.0-rancher1-1 v1.11.9-rancher1-3 v1.11.9-rancher1-2 v1.11.9-rancher1-1 v1.11.8-rancher1-1 v1.11.6-rancher1-1 v1.11.5-rancher1-1 v1.11.3-rancher1-1 v1.11.2-rancher1-2 v1.11.2-rancher1-1 v1.11.1-rancher1-1 v1.10.12-rancher1-1 v1.10.11-rancher1-1 v1.10.5-rancher1-2 v1.10.5-rancher1-1 v1.10.3-rancher2-1 v1.10.1-rancher2-1 v1.10.0-rancher1-1 v1.9.7-rancher2-2 v1.9.7-rancher2-1 v1.9.5-rancher1-1 v1.8.11-rancher2-1], got v1.15.11-rancher1-2
```

**Comment by Oats87** [comment]
I'm running into the same issue:

but with 1.17.x
```
[31m
[1m[31mError: [0m[0m[1mexpected rke_config.0.kubernetes_version to be one of [v1.17.4-rancher1-3 v1.17.4-rancher1-1 v1.17.3-rancher1-1 v1.17.2-rancher1-2 v1.17.2-rancher1-1 v1.17.0-rancher1-2 v1.17.0-rancher1-1 v1.16.8-rancher1-3 v1.16.8-rancher1-1 v1.16.7-rancher1-1 v1.16.6-rancher1-2 v1.16.6-rancher1-1 v1.16.4-rancher1-1 v1.16.3-rancher1-1 v1.16.2-rancher2-1 v1.16.2-rancher1-1 v1.16.1-rancher1-1 v1.15.11-rancher1-3 v1.15.11-rancher1-1 v1.15.11-rancher1-0 v1.15.10-rancher1-1 v1.15.9-rancher1-1 v1.15.7-rancher1-1 v1.15.6-rancher1-2 v1.15.5-rancher2-2 v1.15.5-rancher1-2 v1.15.5-rancher1-1 v1.15.4-rancher1-2 v1.15.4-rancher1-1 v1.15.3-rancher1-1 v1.15.2-rancher1-1 v1.15.0-rancher1-1 v1.14.10-rancher1-1 v1.14.9-rancher1-2 v1.14.9-rancher1-1 v1.14.8-rancher2-1 v1.14.8-rancher1-1 v1.14.7-rancher1-1 v1.14.6-rancher1-1 v1.14.5-rancher1-1 v1.14.3-rancher1-1 v1.14.1-rancher1-2 v1.14.1-rancher1-1 v1.13.12-rancher2-1 v1.13.12-rancher1-1 v1.13.11-rancher1-1 v1.13.10-rancher1-2 v1.13.10-rancher1-1 v1.13.9-rancher1-2 v1.13.9-rancher1-1 v1.13.7-rancher1-1 v1.13.5-rancher1-3 v1.13.5-rancher1-2 v1.13.5-rancher1-1 v1.13.4-rancher1-2 v1.13.4-rancher1-1 v1.13.1-rancher1-2 v1.13.1-rancher1-1 v1.12.10-rancher1-2 v1.12.10-rancher1-1 v1.12.9-rancher1-1 v1.12.7-rancher1-4 v1.12.7-rancher1-3 v1.12.7-rancher1-2 v1.12.7-rancher1-1 v1.12.6-rancher1-2 v1.12.6-rancher1-1 v1.12.5-rancher1-2 v1.12.5-rancher1-1 v1.12.4-rancher1-1 v1.12.3-rancher1-1 v1.12.1-rancher1-1 v1.12.0-rancher1-1 v1.11.9-rancher1-3 v1.11.9-rancher1-2 v1.11.9-rancher1-1 v1.11.8-rancher1-1 v1.11.6-rancher1-1 v1.11.5-rancher1-1 v1.11.3-rancher1-1 v1.11.2-rancher1-2 v1.11.2-rancher1-1 v1.11.1-rancher1-1 v1.10.12-rancher1-1 v1.10.11-rancher1-1 v1.10.5-rancher1-2 v1.10.5-rancher1-1 v1.10.3-rancher2-1 v1.10.1-rancher2-1 v1.10.0-rancher1-1 v1.9.7-rancher2-2 v1.9.7-rancher2-1 v1.9.5-rancher1-1 v1.8.11-rancher2-1], got v1.17.4-rancher1-2[0m

[0m  on main.tf line 1, in resource "rancher2_cluster" "cluster":
   1: resource "rancher2_cluster" "cluster"
```

**Comment by rawmind0** [comment]
Yes, i'm working to release a patch. It's an issue on how tf provider is getting kontainer-metadata.

**Comment by rawmind0** [comment]
Submitted PR #296 to fix the issue

**Comment by rawmind0** [comment]
PR included on release v1.8.2. Please reopen issue if needed.
